Makefile

```diff
@@ -149,6 +151,7 @@ local-install-firefox:
 	install -m 0644 platform/firefox/linux_entra_sso.json ~/.mozilla/native-messaging-hosts
 	sed -i 's|/usr/local/lib/linux-entra-sso/|'$(HOME)'/.mozilla/|' ~/.mozilla/native-messaging-hosts/linux_entra_sso.json
 	install -m 0755 linux-entra-sso.py ~/.mozilla
+	${Q}sed -i $(UPDATE_VERSION_PY) ~/.mozilla/linux-entra-sso.py
```

Comment:
Any particular reason to hide this `sed` but not the one two line above (and the same in the other cases)?

Reply:
Old code vs. new code. Anyways, good catch. I'll clean it up.